.github/workflows/deploy-pr-preview.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy PR Preview
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies and build 
+        run: |
+          yarn install
+          yarn build  # Modify this command based on your build setup
+
+      - name: Deploy PR Preview
+        run: |
```

Comment:
When we test run this it fails.
<img width="1276" alt="image" src="https://github.com/uptane/uptane.github.io/assets/89217603/aeb1ca4b-17b7-4900-aef5-f2fb7c373cfb">